### PR TITLE
Add withdrawn notice for drugs (where relevant)

### PR DESCRIPTION
**Pull request by peatroot**

### CODE REVIEW

**Comment by peatroot** [comment]
Example: CHEMBL101

Screenshot 2019-07-23 at 10 41 13

**Comment by peatroot** [comment]
Moved adjacent to title.

Screenshot 2019-07-23 at 12 08 38

File: src/public/drug/WithdrawnNotice.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      marginTop: 8,
+      border: `2px solid ${theme.palette.secondary.main}`,
+    },
+    header: {
+      padding: '8px 16px',
+      color: 'white',
+      backgroundColor: theme.palette.secondary.main,
+    },
+    content: {
+      padding: '8px 16px !important',
```
Comment (mirandaio) [inline src/public/drug/WithdrawnNotice.js]: I'm guessing the `!important` was necessary?

File: src/public/drug/Page.js [hunk 2]
```diff
@@ -99,19 +109,21 @@ class DrugPage extends Component {
...
-                          type,
-                          tradeNames,
-                          maximumClinicalTrialPhase,
-                          yearOfFirstApproval,
-                          molecularFormula,
-                          description: null,
-                          synonyms,
-                        }}
-                      />
+                      <React.Fragment>
```
Comment (mirandaio) [inline src/public/drug/Page.js]: `Fragment` is not necessary here.

File: src/public/drug/WithdrawnNotice.js (hunk 1 above)
Comment (peatroot) [reply src/public/drug/WithdrawnNotice.js]: Removed.

File: src/public/drug/Page.js (hunk 2 above)
Comment (peatroot) [reply src/public/drug/Page.js]: Removed.

**Pull request merged**